src/javascript/app/pages/trade/purchase/contract-details.jsx

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable import/no-unresolved */
+import React, { useEffect, useState } from 'react';
+import { Button,  Skeleton, Text } from '@deriv-com/quill-ui';
+import { LabelPairedArrowLeftMdRegularIcon } from '@deriv/quill-icons/LabelPaired';
+import ContractTable from './contract-table';
+import Portal from '../../portal';
+import purchaseManager from '../../../common/purchase_manager';
+import { usePurchaseChange } from '../../../hooks/events';
+import { localize } from '../../../../_common/localize';
+import { Explanation } from '../../bottom/explanation';
+import { TimeTooltipWrapper, triggerClick } from '../../../common/helpers';
+
+const AuditSection = ({ data }) => {
+    const auditData = {
+        start: {
+            title  : localize('Contract starts'),
+            content: data?.auditDataStart,
+        },
+        end: {
+            title  : localize('Contract ends'),
+            content: data?.auditDataEnd,
+        },
+        details: {
+            title  : localize('Contract details'),
+            content: data?.auditDataDetails,
+        },
+    };
+
+    return (
+        <>
+            <div className='header-box'>
+                <Button
+                    variant='tertiary'
+                    size='lg'
+                    icon={<LabelPairedArrowLeftMdRegularIcon />}
+                    color='black'
+                    onClick={() => {
+                        purchaseManager.set({
+                            cd_showAudit: false,
+                        });
+                        triggerClick('#contract_purchase_button');
+                    }}
+                />
+                <div className='title-box'>
+                    <Text size='md' bold>
+                        {localize('Audit page')}
+                    </Text>
+                </div>
+                <Button variant='tertiary' size='lg' />
```

Comment:
did you guys try to replace it with a `div` as @balakrishna-deriv  suggested? :)

Reply:
We have updated the other one and missed this one. We are also updating this